server/src/main/java/org/elasticsearch/client/internal/support/AbstractClient.java

```diff
@@ -1581,6 +1582,16 @@ public void resolveIndex(ResolveIndexAction.Request request, ActionListener<Reso
         public ActionFuture<ResolveIndexAction.Response> resolveIndex(ResolveIndexAction.Request request) {
             return execute(ResolveIndexAction.INSTANCE, request);
         }
+
+        @Override
+        public void resolveCluster(ResolveClusterAction.Request request, ActionListener<ResolveClusterAction.Response> listener) {
+            execute(ResolveClusterAction.INSTANCE, request, listener);
+        }
+
+        @Override
+        public ActionFuture<ResolveClusterAction.Response> resolveCluster(ResolveClusterAction.Request request) {
+            return execute(ResolveClusterAction.INSTANCE, request);
+        }
```

Comment:
I think we don't really need these util methods any more, they're something of a holdover from the days of the transport client. Just invoke `client.execute()` on the action type itself, directly, as needed.

Reply:
Fixed in next push.